packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+# 50.14.0 - Month DD, YYYY
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Remove channelService from core extension ([PR #2853](https://github.com/forcedotcom/salesforcedx-vscode/pull/2853))
```

Comment:
This can probably go with under the hood changes as well, it doesn't impact the experience